aio/src/app/custom-elements/code/code.component.ts

```diff
@@ -111,12 +112,13 @@ export class CodeComponent implements OnChanges {
     const leftAlignedCode = leftAlign(this.code);
     this.setCodeHtml(leftAlignedCode); // start with unformatted code
     this.codeText = this.getCodeText(); // store the unformatted code as text (for copying)
-
-    this.pretty
+    const skipPrettify = of(undefined);
```

Comment:
```suggestion

    const skipPrettify = of(undefined);
```

Also, can you please log a warning when specifying linenums with `language === 'none'` (as mentioned in https://github.com/angular/angular/pull/41335#discussion_r600737153)?